frontend/Dockerfile.dev

```diff
@@ -1,5 +1,10 @@
 # Dockerfile for local development
+<<<<<<< HEAD
```

Comment:
Merge conflict resolve needed

Reply:
Oh crap, didn't see that comment. We can fix it in post. 😁